### ISSUE TITLE
Problem with Intervention\Image or \Imagick 

### ISSUE THREAD

**Issue by tobiasvielmetter** [spatie/image-optimizer#25]
After running the ImageOptimizer::optimize and then trying to open it with \Imagick (or Intervention\Image with the imagick driver) I get no php error, but nginx returns a 502 Gateway error.
The file is optimised, but basically opening the optomized jpg file with \Imagick errors fatally.

Does anyone have a idea how to debug this? Nginx does not log any error, even when I switch error reporting onto "info" and laravel only logs the debug from ImageOptimizer, which looks fine (as the image is optomized, I am assuming this is more of a crash issue with \Imagick that cannot handle the optomized image).

Test System: Recent Laravel Homestead, Ubuntu, Nginx, PHP7.1. etc.

**Comment by freekmurze** [comment]
You could take a look at the output the various optimization tools write. That might provided a hint to what's causing the problem.

Here's how to enable logging: https://github.com/spatie/image-optimizer#logging-the-optimization-process

**Comment by tobiasvielmetter** [comment]
@freekmurze Thanks, I am actually using your Laravel Fassade and have logging turned on in the config. The Log looks fine to me, the optimization does not error.
The problem starts when I use the optimized file in Imagick or in Intervention\Image with the imagick driver, which does not throw any error, but halts the process fatally. I will do a bit more testing today to see what the issue is. If I find anything relevant I will write it here. It might be an issue with Imagick digesting images that have been run through Jpegoptim. Will test this some more. Thanks.

**Comment by tobiasvielmetter** [comment]
@freekmurze Ok, after quite a bit of debugging I have found out that the issue was indeed with imagick and the optimised jpg image from Jpegoptim. I also found out that I had an old version of Laravel Homestead running as my test system (vagrant VM). After upgrading the vm to the newest version and installing `sudo apt install php-imagick` again, it works fine. It also worked fine with gd, but that increased the overall filesizes and I dont really like gd and the quality of output, so that was not an option.

So really this is an issue with older imagick versions apparently.

Thanks for this great package btw!
